v1.0/annotation.schema.json

```diff
@@ -0,0 +1,95 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/Annotation",
+  "description": "Structured information describing an annotation.",
+  "required": [
+    "alias",
+    "bestViewPoint",
+    "criteriaType",
+    "internalIdentifier",
+    "laterality",
+    "namingTerm",
+    "visualizedIn"
+  ],
+  "properties": {
+    "alias": {
+      "type": "string",
+      "_instruction": "Enter the user-defined identifier for this annotation (e.g. a short name)."
+    },    
+    "bestViewPoint": {      
+      "_instruction": "Add the coordinate point identifying the best view of this annotation in space.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/sands/CoordinatePoint"
+      ]
+    },
+    "criteria": {
+      "_instruction": "Add the protocol execution defining the criteria that were applied to produce this annotation.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/ProtocolExecution"
+      ]
+    },
+    "criteriaType": {
+      "_instruction": "Add the type of criteria used to define this annotation (processive or asserted).",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/CriteriaType"
+      ]
+    },    
+    "displayColor": {
+      "type": "array",
+      "_instruction": "Enter the preferred display color (RGB) for this annotation.",
+      "items": {
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 255
+      },
+      "minItems": 3,
+      "maxItems": 3
+    },
+    "inspiredBy": {
+      "type": "array",
+      "_instruction": "Add one or several (source) images that inspired the definition of this annotation.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/sands/Image"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },    
+    "internalFileIdentifier": {
+      "type": "string",
+      "_instruction": "Enter the identifier used for this annotation within the file it is stored in."
+    },
+    "laterality": {
+      "type": "array",
+      "_instruction": "Add one or several sides of the body or bilateral organ that this annotation is defined in.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/controlledTerms/Laterality"
+      ],
+      "minItems": 1,
+      "maxItems": 2,
+      "uniqueItems": true
+    },
+    "namingTerm": {
+      "type": "array",
+      "_instruction": "Add one or several anatomical entities (terms) that name this annotation.",
```

Comment:
Very good question. I looked it up in #56. @aeidi89 and I actually suggested to only allow one, but you stated: 

"[...]
- allowing only one entry makes only sense to me if a new terminology version would lead to a new atlas version in which case terminology does not have it's own version similar to annotation, because changes in both will cause to be a new brain atlas version to be defined (which is fine with me) @aeidi89 does this make sense?
  - if this is true the user will only see via the version innovation description what the cause is of the new brain atlas version (new annotations and/or new terminology and/or new referenceSpace);
  - if this is true only in case of referenceSpace a separate version needs to be captured, because the referenceSpace might not be originating from the same project (e.g., Julich-Brain is just using the MNI spaces, but did not created them) 

[...]"

Does that help? 

I think we concluded in the end that it doesn't hurt to all several for now, but that there might be expections for when this isn't logical and should not be done (which the curators need to check for).

Reply:
Conclusion with @lzehl: Keep array for now.